src/vscode-bicep/src/commands/deploy.ts

```diff
@@ -341,13 +381,27 @@ export class DeployCommand implements Command {
         expiresOnTimestamp,
         deployId,
         portalUrl,
+        parametersFileName,
+        parametersFileUpdateOption,
+        updatedDeploymentParameters,
       };
       const deploymentStartResponse: BicepDeploymentStartResponse =
         await this.client.sendRequest("workspace/executeCommand", {
           command: "deploy/start",
           arguments: [deploymentStartParams],
         });
 
+      // If user chose to create a parameters file at the end of deployment flow, we'll
```

Comment:
Comment is confusing - the params file has already been created, we're just opening it.

Actually I think Uche would want it opened whether it was created or updated.

Reply:
@bhsubra Yes let's open it whether they are updating it or creating it for the first time. Generally when changes are made to a file automatically it's good for users to be able to verify the changes are what they expect

Reply:
Updated here to open on all three actions - update, create and overwrite:
https://github.com/Azure/bicep/commit/0688253159e553c00caa73b33545d44d3ef6c3c4